src/main/kotlin/com/sourcegraph/cody/history/HistoryTree.kt

```diff
@@ -170,9 +172,10 @@ class HistoryTree(
   }
 
   private fun getChatsGroupedByPeriod(): Map<String, List<ChatState>> =
-      HistoryService.getInstance()
-          .state
+      HistoryService.getInstance(project)
+          .getHistoryReadOnly()
           .chats
+          .filter { it.messages.isNotEmpty() }
```

Comment:
Is this `.filter { it.messages.isNotEmpty() }` required?

Reply:
Unfortunately, and it's a bit of a hack. I intend to fix it, but I'm not sure how exactly so I think it can stay like that for the time being.

Reply:
Never-mind, looks like there is property ` tree.rowHeight` :O I will fix it.

Reply:
Done.